scss/4-elements/_buttons.scss

```diff
@@ -125,3 +125,16 @@ $button-shadow-size: 4px;
     transform: none;
   }
 }
+
+%cads-button__tertiary--dark {
```

Comment:
Went for a placeholder here as the button is only styled to be dark inside the greedy nav - so this isn't needed as a class ATM.  Can always change later if required.

Reply:
Sounds good; I'd even be tempted to inline this into `cads-header__sign-out` for now.